test/integration/log_checker.go

```diff
@@ -0,0 +1,160 @@
+package integration
+
+import (
+	"bytes"
+	"context"
+	"io"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"regexp"
+	"testing"
+	"time"
+)
+
+type (
+	LogCheck        struct {
+		test			*testing.T
+		clientset 		*kubernetes.Clientset
+		firstCheck		time.Time
+		lastCheck		time.Time
+		since			time.Time
+		logOptions		corev1.PodLogOptions
+		interval		time.Duration
+		timeout			time.Duration
+		namespace 		string
+		podName 		string
+		searching		string
+		sinceLastCheck	bool
+		failFast		bool
+		Err				error
```

Comment:
Same here.

Reply:
ah, github uses really weird tab-size 6.. I guess the thing would be to replace all tabs with spaces to avoid this kind of problems.. However I will do this just in file that I have created in this PR, not to all test files